format.md

```diff
@@ -103,6 +124,17 @@ is a list of xy vertices. The child name of the outer list should be "polygons";
 the child name of the middle list should be "rings"; the child name of the
 inner list should be "vertices".
 
+**Well-known-binary (WKB)**: `Binary` or `LargeBinary` or `FixedSizeBinary`
```

Comment:
Let's leave the addition of WKB (or WKT, or geojson?) for another PR, to focus here on the struct representation?